daft/udf.py

```diff
@@ -30,11 +30,32 @@ class PartialUDF:
     bound_args: inspect.BoundArguments
 
     def expressions(self) -> dict[str, Expression]:
-        return {key: val for key, val in self.bound_args.arguments.items() if isinstance(val, Expression)}
+        parsed_expressions = {}
+        for key, val in self.bound_args.arguments.items():
+            if isinstance(val, Expression):
+                parsed_expressions[key] = val
+            elif isinstance(val, tuple) and all(isinstance(x, Expression) for x in val):
+                for idx, x in enumerate(val):
+                    parsed_expressions[f"{key}{idx}"] = x
```

Comment:
this might lead to a key conflict if we have something like

```
def f(*arg, arg0)
```
we could instead do `f"{key}-{idx}"` or `f"{idx}{key}"` which are impossible to be a arg name.

Reply:
ah, good catch, updated